src/server/rpc/procedures/google-maps/google-maps.js

```diff
@@ -206,9 +206,39 @@ StaticMap.prototype.getLatitude = function(y){
     });
 };
 
+/**
+ * converts image coordinates to earth coordinates.
+ * @param {Number} x x position of the point on the stage
```

Comment:
We can't say "on the stage" because it is technically on the map image. The coordinates are actually relative to the image and not necessarily the stage.